xmppserver/src/main/java/org/jivesoftware/openfire/session/LocalOutgoingServerSession.java

```diff
@@ -413,6 +413,11 @@ else if (ServerDialback.isEnabled() && features.element("dialback") != null) {
             if (connection != null) {
                 connection.forceClose();
             }
+
+            if (JiveGlobals.getBooleanProperty(ConnectionSettings.Server.STRICT_CERTIFICATE_VALIDATION, true)) {
+                log.warn( "Unable to create a new session: not trying dialback as a fallback, as server strictCertificateValidation is enabled" );
```

Comment:
It is better to avoid using the term 'dialback' here. 

When this message is logged, we do not know if Dialback is even enabled. In future versions of Openfire, Dialback support might be removed, or a third authentication mechanism (that we're not naming here) might be added. The message as logged now is based on assumptions, that might not be true in all cases when it is logged. This can add up to a lot of confusion in practice.

Instead, replace this with what's factually being done in the if-statement that is logging the message. Something like:
> Aborting attempt to create outgoing session as TLS handshake failed, and strictCertificateValidation is enabled.

Reply:
fixed